package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "serve": "vite",
+    "serve:prod": "vite --mode production",
```

Comment:
Is it possible that devs may accidentally modify the production data by spinning up a local build that uses production data? 

I don't suspect this happening since only the owner of the data can modify it (i.e the email check in the read/write rules in firestore), but I wanted to ask to make sure its been considered!

Reply:
Yeah it definitely is possible that someone could nuke their own prod data like this, they won't be able to modify anyone else's.

Reply:
Awesome!!